src/service/identity_service.rs

```diff
@@ -96,10 +98,12 @@ impl IdentityServiceApi for IdentityService {
             .map_err(|e| super::Error::Cryptography(e.to_string()))?;
 
         let s = bitcoin::secp256k1::Secp256k1::new();
+        let conf = Config::try_parse();
```

Comment:
If you already have a service container like IdentityService you can just store the Bitcoin network directly on the struct. From there on every method can use the network value direktly via `self.bitcoin_network` and you don't have to parse and handle every time.